CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # Changelog
 
+## [1.12.0] - 2025-03-21
+
+### Added
+
+- Support for any S3 compatible object storage such as MinIO in the `aws_s3` BundlePublisher plugin (#5757)
+- Support for Rego V1 in the authorization policy engine (#5769)
+- Support for SAN-based selectors in the `x509pop` NodeAttestor plugin (#5775)
+
+### Changed
+
+- Timestamps in logs are now formatted to include nanoseconds (#5798)
+- Improved entry lookup performance in NewJWTSVID and BatchNewX509SVID server RPCs (#5819)
+- Internal `Generator` interface in the `keymanagerbase` package was changed to return the more general `crypto.Signer` type (#5843)
+- Increased the maximum number of idle database connections to 100 (#5853)
+- The maximum idle time per database connection is now set to 30 seconds (#5853)
+- Small documentation improvements (#5873, #5876)
+- Agents now use the SyncAuthorizedEntries API for periodically synchronization of authorized entries by default (#5906)
+- The experimental events-based cache now allows reading events from read-only replicas when data staleness is tolerated (#5911)
```

Comment:
how is this affecting users? better performance? may we highlight that?

Reply:
Yes, I think performance is the most important thing here.